sink/sink

```diff
@@ -195,22 +195,23 @@ class GitHub(object):
 
     def push(self, status):
         github = status.get("github", { })
+        http = status.get("github", { })
```

Comment:
I suppose you mean something else here, as otherwise this would just duplicate the entries in `requests`.

Reply:
Thanks. Fixed.